### DISCUSSION TITLE
[Feature Request] Add context to better support custom stylings

### DISCUSSION THREAD

**Discussion by apfelbox** [storyblok/monoblok#277] [Ideas]
[storyblokurl]: https://www.storyblok.com?utm_source=github.com&utm_medium=issue-template

It would be great if the library could add the usage of some kind of context.
For example, I want to add a custom "checkmark" component in every `<li>`, but only if it was nested inside a `<ul>`, not an `<ol>`. It would be great to let resolvers set a flag, that nested resolvers then can use.

---

## Expected Behavior

It would be great if that feature could be added.

## Current Behavior

Not implemented yet.

## Steps to Reproduce

n/a

**Comment by maoberlehner** [comment]
Hey @apfelbox! Thank you for your feature request! Could you provide a concrete code example of how the API for such a feature might look/how you would use it if it existed?
